src/main/scala/edu/berkeley/ce/rockslicing/RockSlicer.scala

```diff
@@ -82,17 +97,21 @@ object RockSlicer {
       // Convert reconstructed blocks to requested output
       if (arguments.toInequalities) {
         val jsonReconBlocks = squeakyCleanRecon.map(Json.blockToMinimalJson)
-        printToFile(new File("reconstructedBlocks.json")) { field =>
-          jsonReconBlocks.foreach(field.println)
+        val pw = new PrintWriter(new File("reconstructedBlocks.json"))
+        jsonReconBlocks.foreach { block =>
+          pw.write(block+"\n")
```

Comment:
Let's use something more like `println(block)` here. This would be a more conventional use of `PrintWriter`. This sure is better than the old code though!